reactivesocket-cpp/src/Frame.cpp

```diff
@@ -114,34 +117,91 @@ std::ostream& operator<<(std::ostream& os, const FrameHeader& header) {
   std::bitset<16> flags(header.flags_);
   return os << header.type_ << "[" << flags << ", " << header.streamId_ << "]";
 }
+
+constexpr auto MAX_META_LENGTH = std::numeric_limits<int32_t>::max();
```

Comment:
static constexpr auto kMAX_META_LENGTH
constants should start with k-prefix
symbols which are not relevant outside of the compilation unit should be declared as static